spyderlib/config.py

```diff
@@ -477,6 +477,7 @@ def is_ubuntu():
               '_/save current layout': "Shift+Alt+S",
               '_/toggle default layout': "Shift+Alt+Home",
               '_/layout preferences': "Shift+Alt+P",
+              '_/restart': "Shift+Ctrl+R",

```

Comment:
Shift+Ctrl+Alt+AltGr+Meta+R ?
A bit more seriously, the other shortcuts use Shift+Alt, so it should be OK.